### DISCUSSION TITLE
verse_onprem/3.2/admin/enabling_domi_online_meetings

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#69] [Q&A]
# verse_onprem/3.2/admin/enabling_domi_online_meetings

Beginning with Verse 3.2.1, administrators can apply a notes.ini setting to control the meeting services that are available for users to schedule dynamic online meetings.

https://help.hcltechsw.com/verse_onprem/3.2/admin/enabling_domi_online_meetings.html?hl=domi

**Comment by chuckdumont** [comment]
Anoter note to add:

**Note:** this feature requires that the Domino server be able to send outgoing HTTPS messages to the following external resources.
* For all providers - https://integration-auth-token.hcltechsw.com
* For Webex - https://webexapis.com/v1/meetings
* For MS Teams - https://graph.microsoft.com/v1.0/me/onlineMeetings
* For GoToMeetings - https://api.getgo.com/G2M/rest/meetings
* For Zoom - https://api.zoom.us/v2/users/me/meetings

You may need to adjust your proxy/firewall rules to allow this.
